### DISCUSSION TITLE
More JSdoc

### DISCUSSION THREAD

**Discussion by MarlonPassos-git** [rayepps/radash#120] [Ideas]
one thing I really like about the new code editors is when hovering over the function it shows a lot of useful information about the current function.

In my mind it would be really cool to add a link to the documentation and at least one example along with the function.

of course this would have the effect of augmenting the file a little with jsdoc, but it would make everything more documented

If I find something valid I can be making a pr by adding both @link and @example 

![image](https://user-images.githubusercontent.com/63638196/197365547-dfb64a78-397e-4932-a613-6aa161ddd433.png)

![image](https://user-images.githubusercontent.com/63638196/197365557-970c178d-c15e-40ae-97c1-15db6fa6a8e4.png)

then you would have to know which phrase to use to indicate documentation
- see documentation
- documentation
- see alphabet documentation

**Comment by sodiray** [comment]
I'm a big fan of better docs and DX. If there's anyway I can help let me know :)
